### ISSUE TITLE
API fails downloading the Island of Guam

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1626]
**[Submitted to the original trac issue database at 11.18am, Saturday, 28th February 2009]**

curl http://www.openstreetmap.org/api/0.5/ma . 3428748554

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<body>
  <h1>File not found</h1>
  <p>Change this error message for pages not found in public/404.html</p>
</body>
</html>curl: (6) Could not resolve host: ; Host not found
curl: (6) Could not resolve host: 3428748554; Host not found

**Comment by tomhughes** [comment]
**Author: tom@compton.nu**
**[Added to the original trac issue at 11.21am, Saturday, 28th February 2009]**

Your URL appears to be completely bogus, which is the cause of your problem.

What has happened is that curl has tried to download http://www.openstreetmap.org/api/0.5/ma which is not a valid URL so has produced a 404 response.

It has then tried to download 3428748554 which it seems to have interpreted as a hostname (presumably it decided to treat it as http://3428748554/) which is couldn't resolve.

**Comment by tomhughes** [comment]
**Author: charris**
**[Added to the original trac issue at 1.19pm, Saturday, 28th February 2009]**

Sorry, I pasted the wrong URL..

When I view the map at

http://www.openstreetmap.org/?lat=13.495&lon=144.832&zoom=10&layers=B000FTF

I selected a small area and with an export XML, it will return "500 - Internal Server Error"

http://api.openstreetmap.org/api/0.5/map?bbox=144.8161,13.4568,144.8909,13.5493

Curl returns the same error.

For other areas for Guam, it can be selected.

----------------------------------------------------

If I use JOSM, and select the whole region, I get the following error...

Server returned internal error.  Try a reduced area or retry after waiting some time.
